src/nvim/terminal.c

```diff
@@ -404,7 +404,7 @@ void terminal_enter(void)
 
   // Disable these options in terminal-mode. They are nonsense because cursor is
   // placed at end of buffer to "follow" output. #11072
-  win_T *save_curwin = curwin;
+  handle_T save_handle = curwin->handle;
```

Comment:
nit: I'd leave the name as it was, it's a bit more semantically descriptive

Reply:
Rebased & done

Reply:
Note that there is a `win_T *save_curwin = curwin` in the same file later on, so it might get confusing. That's why I renamed it in the first place.